### PR TITLE
Match .env vars with console-generated names

### PR DESCRIPTION
**Pull request by ronenh**
This PR renames environment variables to match the names in the `.env` file generated by the console in the getting started and Quickstart apps.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,10 +2,10 @@ FLASK_RUN_PORT=3001
 FLASK_APP=server.py
 
 JWKS_URI=https://citadel.demo.aserto.com/dex/keys
-OIDC_ISSUER=https://citadel.demo.aserto.com/dex
-OIDC_CLIENT_ID=citadel-app
+ISSUER=https://citadel.demo.aserto.com/dex
+CLIENT_ID=citadel-app
```
Comment (ogazitt) [inline .env.example:6]: This should be AUDIENCE

File: .env.example [hunk 2]
```diff
@@ -23,5 +23,5 @@ DIRECTORY_GRPC_CERT_PATH=$HOME/.config/topaz/certs/grpc-ca.crt
 # ASERTO_TENANT_ID={Your Aserto Tenant ID guid}
 # ASERTO_AUTHORIZER_API_KEY={Your Authorizer API Key}
 # ASERTO_DIRECTORY_API_KEY={You Directory (read-only) API Key}
-# ASERTO_POLICY_INSTANCE_NAME=policy-todo
-# ASERTO_POLICY_INSTANCE_LABEL=policy-todo
+# ASERTO_POLICY_INSTANCE_NAME=todo-rebac
```
Comment (ogazitt) [inline .env.example:26]: we don't yet have everything plumbed to todo-rebac.

so in the immediate term this should be ASERTO_POLICY_INSTANCE_NAME=todo

File: .env.example [hunk 3]
```diff
@@ -23,5 +23,5 @@ DIRECTORY_GRPC_CERT_PATH=$HOME/.config/topaz/certs/grpc-ca.crt
...
+# ASERTO_POLICY_INSTANCE_LABEL=todo-rebac
```
Comment (ogazitt) [inline .env.example:27]: ditto

**Review by ogazitt — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (ronenh) [reply .env.example]: Good catch! 🏆

**Review by ogazitt — APPROVED** [review]

**Pull request merged**